lib/datadog/tracing/distributed/trace_context.rb

```diff
@@ -42,7 +42,9 @@ def extract(data)
 
           return unless trace_id # Could not parse traceparent
 
-          tracestate, sampling_priority, origin, tags, unknown_fields = extract_tracestate(fetcher[@tracestate_key])
+          tracestate, sampling_priority, origin, dd_parent_id, tags, unknown_fields = extract_tracestate(
```

Comment:
```suggestion
          tracestate, sampling_priority, origin, parent_id, tags, unknown_fields = extract_tracestate(
```
I think it's OK to shorten it, without the `dd_` prefix, because everything in this file tends to be `dd` anyway.